### DISCUSSION TITLE
Is There a Way to Link Password Authentication After Google Authentication?

### DISCUSSION THREAD

**Discussion by dninomiya** [supabase/supabase#29909] [Questions] [answered]
The documentation explains how to link Google authentication to an existing user, but I can’t figure out how to enable password authentication later for a user who initially authenticated via Google. Is this possible?

https://supabase.com/docs/guides/auth/auth-identity-linking?queryGroups=language&language=js

**Comment by GaryAustin1** [comment, marked as answer]
I believe you just updateUser to add a password.  At one time I thought there was a doc section on it in the linking section but it has been redone.
This mentions it:  https://supabase.com/blog/supabase-auth-identity-linking-hooks

**Reply by dninomiya** [reply]
Solved! thank you for your support!
